apple/internal/partials/bitcode_symbols.bzl

```diff
@@ -62,8 +62,8 @@ def _bitcode_symbols_partial_impl(
             # Get the UUID of the arch slice and use that to name the bcsymbolmap file.
             copy_commands.append(
                 ("cp {bitcode_file} " +
-                 "${{OUTPUT_DIR}}/$(dwarfdump -u -arch {arch} {binary} " +
-                 "| cut -d' ' -f2).bcsymbolmap").format(
+                 "${{OUTPUT_DIR}}/$(dwarfdump -u {binary} " +
+                 "| grep \"({arch})\" | cut -d' ' -f2).bcsymbolmap").format(
```

Comment:
I bet `dwarfdump-classic` would work here. I think the problem is they switched over to `llvm-dwarfdump`.

Reply:
Interesting! It looks like 10.1 doesn't have that though so we'd have to switch on that

Reply:
> It looks like 10.1 doesn't have that

🤦‍♂️ 

This is nicer anyway, was just pointing out what likely caused the break.